auth/src/express/middleware/parser.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { Logger } from '../../api/services/logger';
+
+export namespace Parser {
```

Comment:
Since this is a middleware, you either just write a middleware (and not name it `Parser`) or separate is clearly: First, you have a parser,but the parser must not know about a request, response, ... Second, you have the middleware, which just does something like this:
```
try {
    req.body = parser.ensureValidJson(req.body);
} catch (e) {
    //return a 400er -> The req.body was not valid. What is "valid" is defined by the parser
}
```